cypress/e2e/rooms/changeRoomsPermission/changeRoomPermissionViewerToOwner.feature

```diff
@@ -0,0 +1,88 @@
+@regression_test
+@stable_test
+Feature: Room - Change room permission (Viewer - Owner)
+
+    As a teacher, I want to change a participant’s room permission from viewer to owner, so that other users can access the room and leave it as needed
+
+    Scenario: Teacher change room permission of other participants, including pre-conditions
+        Given I am logged in as a '<teacher_2>' at '<namespace>'
+        Given I am logged in as a '<teacher_1>' at '<namespace>'
+
+        # pre-condition: first teacher creating a new room
+        Given a room named '<room_name>' exists
+
+        # first teacher is able to add participants
+        Then I see the detail page of room '<room_name>'
+        When I click on three dot menu in room page
+        When I select the three dot menu action 'room-members'
+        Then I see the page Edit participants of room '<room_name>'
+        When I click on FAB to add participants
+        Then I see modal Add participants
+        Then I see school '<school_name>' in dropdown School
+        Then I see role '<role_name>' in dropdown Role
+        When I enter '<participant_name>' in dropdown Name
+        When I select the first name from the dropdown
+        When I click on the button Add participant
+        Then I see '<participant_name>' in the room participants list
+
+        # first teacher changes role for other user
+        When I click on button Three Dot Menu to add participant '<participant_name>'
+        Then I see button Change Role Permission is visible
+        When I click on button 'Change-Permission' in the sub-menu
+        Then I see dialog box Change Role Permission is visible
+        When I change second user role to 'owner'
+        Then I see Role changed to 'owner' for second user
+        Then I click on button 'Confirm' in the 'Role' action menu
+        Then I click on button 'Confirm' in the 'Owner' action menu
+
+        # second user logs in, accesses the room, verifies that restricted functionalities are not visible to them, and then leaves the room
+        Given I am logged in as a '<teacher_2>' at '<namespace>'
+        When I go to room overview
+        Then I see '<room_name>' on room overview page
+        When I go to room '<room_name>'
+        Then I see the detail page of room '<room_name>'
+        Then I see button Fab Create Room Board
+        When I click on three dot menu in room page
+        When I select the three dot menu action 'edit'
+        When I enter the room name '<room_name_edited>'
+        When I click on the button Save room
+        Then I see the detail page of room '<room_name_edited>'
+        When I click on three dot menu in room page
+        When I select the three dot menu action 'room-members'
+        Then I see the page Edit participants of room '<room_name_edited>'
+        Then I see button Fab Add Member
+        Then I see info text
```

Comment:
Shouldn't this info text be detailed, about which text (as parameter)?

Reply:
done